mappings/net/minecraft/world/biome/BuiltInBiomes.mapping

```diff
@@ -0,0 +1,3 @@
+CLASS net/minecraft/class_1972 net/minecraft/world/biome/BuiltInBiomes
```

Comment:
Currently there's the existing `BuiltinRegistries`- should this capitalisation match that?

Reply:
If anything, `BuiltinRegistries` should be changed to `BuiltInRegistries`. 'Builtin' is not a single word.

Reply:
Vanilla text uses `built-in`, which is one hyphenated word.